### DISCUSSION TITLE
commit-activity hover item left over when navigating back to Pulse

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#16052] [Other Feature Feedback, Questions, & Ideas]
Steps:

1. Visit https://github.com/apple/swift/pulse
2. Click `Commits` on the left
3. Hovering over a graph item for `08/01`:
    <img width="1370" alt="image" src="https://user-images.githubusercontent.com/2119212/166748294-06b0e06e-4dbf-4dd9-831c-2c28fc19d066.png">
4. Use cmd-back on the keyboard to switch back to the Pulse view:
    <img width="1332" alt="image" src="https://user-images.githubusercontent.com/2119212/166748108-d4837695-a3cb-4409-a38a-23ca62e48b77.png">

**Comment by queenofcorgis** [comment]
Hi @jsoref thanks for posting! Would you mind elaborating on what you're experiencing? Is it unexpected behavior?

**Reply by jsoref** [reply]
The `172 commits` black info bubble in the second picture shouldn't be there. It's a carryover dom object from the other view.
